src/azure-cli/azure/cli/command_modules/role/_params.py

```diff
@@ -263,7 +264,6 @@ def load_arguments(self, _):
     with self.argument_context('ad user') as c:
         c.ignore('_subscription')
         c.argument('mail_nickname', help='mail alias. Defaults to user principal name')
-        c.argument('force_change_password_next_login', arg_type=get_three_state_flag(), help='Require the user to change their password the next time they log in. Only valid when --password is specified')
```

Comment:
May I aks removing this parameter looks like a breaking change, right? If so, why not indicate it in the PR title?

Reply:
Good question! `force_change_password_next_login` has been renamed to `force_change_password_next_sign_in`. `force_change_password_next_sign_in` is no longer used.

Reply:
Got it, thanks